### ISSUE TITLE
Log level regression

### ISSUE THREAD

**Issue by h0tw1r3** [fluent/fluent-bit#225]
Log level configuration is completely ignored.
Built with trace support, set level but no trace output.

Reverting #218 fixes the regression.

**Comment by edsiper** [comment]
Thanks for reporting this issue.  I've done two tests with version 0.11 and found the verbosity level problem is only raised when using a configuration file, command line works as expected.

I will make sure to include a fix for 0.11.1 release.

# tests

## Command Line

### Command line (no verbosity)

```
$ bin/fluent-bit -i cpu -o exit -f 1
Fluent-Bit v0.11.0
Copyright (C) Treasure Data

[2017/04/03 19:04:40] [ info] [engine] started
[2017/04/03 19:04:41] [ info] [input] pausing cpu.0
[2017/04/03 19:04:41] [ warn] [engine] service will stop in 5 seconds
[2017/04/03 19:04:45] [ info] [engine] service stopped
```

### Command Line (verbosity = debug)
```
$ bin/fluent-bit -i cpu -o exit -f 1 -v
Fluent-Bit v0.11.0
Copyright (C) Treasure Data

[2017/04/03 19:04:48] [ info] [engine] started
[2017/04/03 19:04:48] [debug] [router] default match rule cpu.0:exit.0
[2017/04/03 19:04:48] [debug] [input cpu.0] [mem buf] size = 327
[2017/04/03 19:04:49] [debug] [task] created task=0x55637be87180 id=0 OK
[2017/04/03 19:04:49] [ info] [input] pausing cpu.0
[2017/04/03 19:04:49] [ warn] [engine] service will stop in 5 seconds
[2017/04/03 19:04:49] [debug] [task] destroy task=0x55637be87180 (task_id=0)
[2017/04/03 19:04:53] [ info] [engine] service stopped
```

### Command Line (verbosity = trace)

```
$ bin/fluent-bit -i cpu -o exit -f 1 -vv
Fluent-Bit v0.11.0
Copyright (C) Treasure Data

[2017/04/03 19:04:55] [ info] [engine] started
[2017/04/03 19:04:55] [debug] [router] default match rule cpu.0:exit.0
[2017/04/03 19:04:55] [trace] [in_cpu] CPU 4.25%
[2017/04/03 19:04:55] [debug] [input cpu.0] [mem buf] size = 327
[2017/04/03 19:04:56] [trace] [task 0x5562c4b58180] created (id=0)
[2017/04/03 19:04:56] [debug] [task] created task=0x5562c4b58180 id=0 OK
[2017/04/03 19:04:56] [trace] [engine dispatch] task #0 created 0x5562c4b58180
[2017/04/03 19:04:56] [trace] [thread 0x5562c4b58220] created (custom data at 0x5562c4b58248, size=64
[2017/04/03 19:04:56] [ info] [input] pausing cpu.0
[2017/04/03 19:04:56] [trace] [engine] flush enqueued data
[2017/04/03 19:04:56] [ warn] [engine] service will stop in 5 seconds
[2017/04/03 19:04:56] [trace] [engine] [task event] task_id=0 thread_id=0 return=OK
[2017/04/03 19:04:56] [trace] [thread] destroy thread=0x5562c4b58220 data=0x5562c4b58248
[2017/04/03 19:04:56] [debug] [task] destroy task=0x5562c4b58180 (task_id=0)
[2017/04/03 19:05:00] [ info] [engine] service stopped
```

**Comment by edsiper** [comment]
I have pushed a fix in 0.11 branch, the solution will be available on version 0.11.1

thanks again for reporting this, I am closing this issue for now.
